addons/rmsmartshape/shapes/shape_base.gd

```diff
@@ -1328,8 +1276,8 @@ func _edge_data_get_tess_point_count(index_map: SS2D_IndexMap) -> int:
 		if this_idx > next_idx:
 			count += 1
 			continue
-		var this_t_idx: int = get_tessellated_idx_from_point(points, t_points, this_idx)
-		var next_t_idx: int = get_tessellated_idx_from_point(points, t_points, next_idx)
+		var this_t_idx: int = point_idx_to_t_points_idx[this_idx][0]
+		var next_t_idx: int = point_idx_to_t_points_idx[next_idx][0]
 		var delta: int = next_t_idx - this_t_idx
 		count += delta
 	return count
```

Comment:
This function can take some time and in my manual tests, the result is always equal to `t_points.size() - 1`. If I change to this code, it's faster but the unit tests fail.